### PR TITLE
update js file

### PR DESCRIPTION
**Pull request by peteronyam**
This pull request updates the Readme file, and styles.css, and also adds a JavaScript file to preserve data  to enhance the project's readability and provide more comprehensive information about the project's purpose and goals. The changes aim to help new contributors understand the project and how to contribute to it by adding installation instructions, troubleshooting steps, and a step-by-step guide for building and running the project.

### CODE REVIEW

**Review by aamirkhan2478 — COMMENTED** [review]
Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :red_circle:  :red_circle:  :red_circle: 

You have submitted a project, that hasn't any commit from your partner kindly contact your partner tell for commit because this milestone is a pair programming milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by franclobo — COMMENTED** [review]
Hi @peteronyam, 

terminator

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is a pair-programming project but there are not commits from your partner, if you have been authorized by Student Success to work alone on this Block please paste the email on the PR, similar to if you weren't paired for this block paste an image from your dashboard.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -56,3 +56,22 @@ form.addEventListener('submit', (event) => {
...
+  const fullName = fullNameInput.value;
+  const email = eInput.value;
+  const message = messageInput.value;
+
+  localStorage.setItem('fullName', fullName);
+  localStorage.setItem('email', email);
+  localStorage.setItem('message', message);
+
+  form.reset();
+});
```
Comment (andersonlebon) [inline main.js:77]: - [ ] Kindly make sure when the form is submitted to the email the way it was before and when I refresh the page the data served previously needs to be populated in the form. Below you will find a screencast that gives you more clarification

https://user-images.githubusercontent.com/65068771/233765325-9e30965a-308b-4d5c-924a-a3c32532e42e.mov

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @peteronyam  team

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ Data is preserved correctly 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**